POSTGRES.md

```diff
@@ -0,0 +1,56 @@
+<img src="./images/daml-enterprise-logo.svg" width="400px">
+
+[![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/digital-asset)](https://artifacthub.io/packages/search?repo=digital-asset)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](./LICENSE)
+
+# Postgresql database requirements
+
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Versions](#versions)
+- [Network connection](#network-connection)
+- [License](#license)
+
+## Introduction
+
+This document intends to describe the required postgres database setup as dependency for daml-helm-charts deployment.
+
+## Versions
+
+The following postresql major versions are tested and supported by our components:
+- 11.x
+- 12.x
+- 13.x
+- 14.x
+
+## Network connection
+
+[Max connections](https://www.postgresql.org/docs/14/runtime-config-connection.html#GUC-MAX-CONNECTIONS) setting of Postgresql must be set according to the number of components installed.
+A minimal reference installation using the default values in this repository in all helm charts requires at least 140 connections. 
```

Comment:
We should explain what a base/reference deployment of all helm charts contains:
* Domain with two sequencers
* Participant
* HTTP JSON API service
* Trigger service

We should explain there settings to adjust the max connection use by client pools in the all the helm charts.

We should advise to add a safety margin

Reply:
Point 1 and 3 added.

On point 2, do you mean that we should add the max connections documentation to each of our helm charts README files so that its easier for users to find it?

Reply:
Explain that we have these settings around:
* For Canton helm charts (Domain or Participant), we can set `storage.maxConnections`
* For Daml helm charts (HTTP JSON API and Trigger), we can set `storage.minIdle` and `storage.poolSize`
(We should probably normalize these names to have the same terminology everywhere)